src/contracts/pods/EigenPod.sol

```diff
@@ -42,6 +42,9 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     /// @notice Maximum "staleness" of a Beacon Chain state root against which `verifyBalanceUpdate` may be proven.
     uint256 internal constant VERIFY_BALANCE_UPDATE_WINDOW_SECONDS = 4.5 hours;
 
+    /// @notice Maximum "staleness" of a Beacon Chain state root against which `verifyWithdrawalCredential` may be proven.
+    uint256 internal constant VERIFY_WITHDRAWAL_CREDENTIAL_WINDOW = 7 days;
+
```

Comment:
why is this a different value than `VERIFY_BALANCE_UPDATE_WINDOW_SECONDS` ?

Reply:
My idea is that effective balance changes far less often so we should make it more lenient

Reply:
verifyBalanceUpdate updates the balance with the actual balance

Reply:
> My idea is that effective balance changes far less often so we should make it more lenient

aren't we using effective balance for all balance changes?
`verifyWithdrawalCredentials` also "performs a balance update"
I'm not seeing how these two cases are concretely different / why the window should be different for these at all

Reply:
> aren't we using effective balance for all balance changes? 

Not quite, so when we prove the initial balance, we use effective balance (for efficiency since WC and effective balance are in same container).  But for balance updates, we allow it to be updated to actual balance, which is much better for accuracy

Reply:
My idea is that because its effective balance initially, its ok to allow a more lenient window.

Reply:
> > aren't we using effective balance for all balance changes?
> 
> Not quite, so when we prove the initial balance, we use effective balance (for efficiency since WC and effective balance are in same container). But for balance updates, we allow it to be updated to actual balance, which is much better for accuracy

ahhh, OK, I missed this subtlety, need to make sure this is well documented

Reply:
> My idea is that because its effective balance initially, its ok to allow a more lenient window.

even understanding this difference, is there a requirement for making this window ~28x as lenient? If we expect to have regular state roots available to prove against, I still can't justify this being longer.